definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-deemed-and-dispensed-prod.json

```diff
@@ -5,5 +5,19 @@
     "CaseStateID": "awaitingServicePayment",
     "UserRole": "caseworker-divorce-courtadmin_beta",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "27/07/2020",
```

Comment:
I think these shouldn't be here. Otherwise, these users will be able to see these in the filter.

Reply:
Can they use it if it's "R"?

Reply:
When I removed it we get unit tests failures:

```
1) UserRole authorisations for CaseState
       should allow minimum R access for all Case States per User Role 
         (prod):

      AssertionError: expected 6 to deeply equal 8
      + expected - actual
```

Reply:
That's because something is in the core file, when it should actually be in the named feature-toggle file.